### PR TITLE
Add TensorFlow 2.3 GPU and TPU tests.

### PR DESCRIPTION
**Pull request by aman2930**

### CODE REVIEW

File: tests/tensorflow/r2.3/common.libsonnet [hunk 1]
```diff
@@ -18,8 +18,25 @@ local common = import "../common.libsonnet";
  ModelGardenTest:: common.ModelGardenTest {
```
[the remark this answers is not in the document]
Comment (aman2930) [reply tests/tensorflow/r2.3/common.libsonnet:18]: No worries.

[the remark this answers is not in the document]
Comment (aman2930) [reply tests/tensorflow/r2.3/common.libsonnet:18]: I will add it in subsequent PR.

**Comment by aman2930** [comment]
> Overall LGTM. I have some concern that adding too many nightly functional tests may cause the test suite to run too long with our current capacity. If that's the case we can reduce the frequency of some tests (e.g. alternate nightly and release tests for TF each day) or increase our TPU quota (capacity permitting). How frequently do we need to run functional tests for the release? AFAIK there's at most one change per week as the RCs come out.
>> No. There are more changes. Based on failures or issues in our tests and in the Kokoro tests of TF team, model garden is updated more frequently. That needed tests to run again on that. So, I would suggest to initially keep functional tests for running daily, and based on the output if some tests are passing then we can reduce their frequency, if no change is expected in those.

File: tests/tensorflow/r2.3/keras-api.libsonnet [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      |||
+        export PATH=$PATH:/root/google-cloud-sdk/bin
+        gcloud source repos clone tf2-api-tests --project=xl-ml-test
+        cd tf2-api-tests
+        pip3 install behave
+        behave -e ipynb_checkpoints --tags=-fails -i %s
+      ||| % self.testFeature
+    ),
+    regressionTestConfig: {
+      alert_for_failed_jobs: false,
```
[the remark this answers is not in the document]
Comment (gagika) [reply tests/tensorflow/r2.3/keras-api.libsonnet:35]: I believe so, let's enable alerting.

**Comment by aman2930** [comment]
Alerting enabled.

**Pull request merged**